README.md

```diff
@@ -4,7 +4,7 @@
 
 yDaemon is the next-gen API for Yearn. Based on the API from [yearn-exporter](https://github.com/yearn/yearn-exporter), it brings a lot of new features and benefits without breaking the existing system.
 
-See documentation here: https://ydaemon.ycorpo.com/
+SEE documentation here:  https://ydaemon.yearn.farm/
```

Comment:
Why is see capped? 
Also there is an extra space in the link

Reply:
The changes you highlighted were not intentional, only the link change. I made the necessary adjustments. Thanks!  ;)